tests/wpt/mozilla/meta/mozilla/preferences.html.ini

```diff
@@ -1,10 +1,3 @@
 [preferences.html]
   type: testharness
-  prefs: [
-    dom.testbinding.preference_value.falsy:false,
-    dom.testbinding.preference_value.truthy:true,
-    dom.testbinding.preference_value.string_empty:,
-    dom.testbinding.preference_value.string_test:test,
-    dom.testbinding.preference_value.space_string_test:test1 test2,
-    dom.testbinding.preference_value.quote_string_test:"test1 test2",
-    ]
+  prefs: [dom.testbinding.preference_value.falsy:false, dom.testbinding.preference_value.truthy:true, dom.testbinding.preference_value.string_empty:, dom.testbinding.preference_value.string_test:test, dom.testbinding.preference_value.space_string_test:test1 test2, dom.testbinding.preference_value.quote_string_test:"test1 test2"]
```

Comment:
The thing is that I made the edit to format the ini file when I added the prefs in another PR, but it seems update-wpt format it in 1 line, maybe we should let it that way.